contrib/packs/actions/create.meta.yaml

```diff
@@ -0,0 +1,41 @@
+---
+  name: "create"
+  runner_type: "action-chain"
+  description: "Creates a new pack"
+  enabled: true
+  entry_point: "workflows/create.yaml"
+  parameters:
+    name:
+      type: "string"
+      required: true
+      description: "Name of the pack to create"
+    description:
+      type: "string"
+      description: "Pack description"
+    keywords:
+      type: "array"
+      items:
+        type: "string"
+      description: "Search keywords"
+    version:
+      type: "string"
+      description: "Initial version of the pack"
+      required: true
+    author:
+      type: "string"
+      description: "Pack's author"
+      required: true
+    email:
+      type: "string"
+      description: "Pack's author email"
+    directories:
```

Comment:
Yeah, I mean the rest: directories, READMEs, sample files, etc. A sample config schema would be useful, too. In the future, even more stuff. Pack.yaml can (and even should) be generated from a template. :)